src/opera/util/geo_utils.py

```diff
@@ -86,8 +86,20 @@ def translate_utm_bbox_to_lat_lon(bbox, epsg_code):
     elevation = 0
     xmin, ymin, xmax, ymax = bbox
 
-    lat_min, lon_min, _ = transformation.TransformPoint(xmin, ymin, elevation)
-    lat_max, lon_max, _ = transformation.TransformPoint(xmax, ymax, elevation)
+    ulc = (xmin, ymax)
```

Comment:
Let's include a comment here about the approach being taken and why its needed (correctly handle high-latitude tiles etc..)